src/main/java/com/powsybl/caseserver/CaseController.java

```diff
@@ -224,4 +225,13 @@ public ResponseEntity<List<CaseInfos>> getMetadata(@RequestParam("ids") List<UUI
         LOGGER.debug("get Cases metadata");
         return ResponseEntity.ok().body(caseService.getMetadata(ids));
     }
+
+    @GetMapping(value = "/cases/caseBaseName")
+    @Operation(summary = "Get case base name")
+    @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "case base name"), })
+    public ResponseEntity<String> getCaseBaseName(@RequestParam("caseName") String caseName) {
+        LOGGER.debug("getCaseBaseName request received");
```

Comment:
`    LOGGER.debug("Fetch base name for case: {}", caseName);`

Reply:
Logger only take one String parameter !
  And i am not sûre we are fetching something here ! it is server side

Reply:
but it is server side we are not fetching anything here !

Reply:
It was a suggestion to add LOGGER.debug, and you can choose the appropriate message.